api/request.go

```diff
@@ -185,31 +185,8 @@ func (s SliceRequest) toString() (string, error) {
 type AttributeRequest struct {
 	RequestedResource
 
-	// Horizon / height-map
-	Horizon [][]float32 `json:"horizon" binding:"required"`
-
-	// Rotation of the X-axis (East), counter-clockwise, in degrees
-	Rotation *float32 `json:"rotation" binding:"required"`
-
-	// X-coordinate of the origin
-	Xori *float32 `json:"xori" binding:"required"`
-
-	// Y-coordinate of the origin
-	Yori *float32 `json:"yori" binding:"required"`
-
-	// X-increment - The physical distance between columns in horizon
-	Xinc float32 `json:"xinc" binding:"required"`
-
-	// Y-increment - The physical distance between rows in horizon
-	Yinc float32 `json:"yinc" binding:"required"`
-
-	// Any sample in the input horizon with value == fillValue will be ignored
-	// and the fillValue will be used in the amplitude map.
-	// I.e. for any [i, j] where horizon[i][j] == fillValue then
-	// output[i][j] == fillValue.
-	// Additionally, the fillValue is used for any point in the horizon that
-	// falls outside the bounds of the seismic volume.
-	FillValue *float32 `json:"fillValue" binding:"required"`
+	// Surface along which data must be retrieved
+	Surface core.RegularSurface `json:"surface" binding:"required"`
```

Comment:
This breaks the API. Are we sure we want that? We would have to inform all users and update all demo scripts.

Reply:
Ignore the above comment, I remember why this is _needed_ for future API additions. However, I'm missing some motivation for this change in the commit and also more explicitness on the fact that we are breaking the API here, not just combining some structs internally